creator-node/src/services/prometheusMonitoring/prometheus.constants.js

```diff
@@ -41,12 +44,27 @@ MetricNames = Object.freeze(
 
 const MetricLabels = Object.freeze({
   [MetricNames.ISSUE_SYNC_REQUEST_MONITORING_DURATION_SECONDS_HISTOGRAM]: {
+    // The type of sync issued -- manual or recurring
+    syncType: [_.snakeCase(SyncType.Manual), _.snakeCase(SyncType.Recurring)],
     // The reason another sync is needed
     reason_for_additional_sync: [
       'secondary_progressed_too_slow', // The secondary sync went through, but its clock value didn't increase enough
       'secondary_failed_to_progress', // The secondary's clock value did not increase at all
       'none' // No additional sync is required -- the first sync was successful
     ]
+  },
+  [MetricNames[
+    `STATE_MACHINE_${JOB_NAMES.UPDATE_REPLICA_SET}_JOB_DURATION_SECONDS_HISTOGRAM`
+  ]]: {
+    // Whether or not the user's replica set was updated during this job
+    issuedReconfig: ['false', 'true'],
+    // The type of reconfig, if any, that happened during this job (or that would happen if reconfigs were enabled)
+    reconfigType: [
+      'one_secondary', // Only one secondary was replaced in the user's replica set
+      'multiple_secondaries', // Both secondaries were replaced in the user's replica set
+      'primary_and_or_secondaries', // A secondary gets promoted to new primary and one or both secondaries get replaced with new random nodes,
+      'null' // No change was made to the user's replica set because the job short-circuited before selecting or was unable to select new node(s)
```

Comment:
also i don't see these used anywhere?

Reply:
they're returned in the metric from the update-replica-set job processor (`_determineNewReplicaSet()`), and processJob defaults the metric to the null category if nothing is provided. I think this null category is OK because it reflects the logic of the processor (i.e., it tries to replace 1 or 2 replicas, and if there are 3 then it returns null -- there shouldn't be any other options/categories possible)

Reply:
cool